schema/transportation/segment.yaml

```diff
@@ -308,6 +309,30 @@ properties:
         - dirt
         - paving_stones
         - metal
+    routes:
+      description: Routes this segment belongs to
+      type: array
+      items:
+        type: object
+        unevaluatedProperties: false
+        allOf:
+          - { "$ref": "../defs.yaml#/$defs/propertyContainers/geometricRangeScopeContainer" }
+        properties:
```

Comment:
Is there an minimal validation we can offer on any of these sub-properties?

For *e.g.*, is anything `required`? (Hopefully something.) And is there any other validation? We can usually at least do `minLength: 1` and `pattern: ^(\S.*)?\S$` ([example](https://github.com/OvertureMaps/schema/blob/7c9c6e7c807120345812b2a76d264b890d4bfc0f/schema/defs.yaml#L204))